src/components/TodoErrorList/TodoErrorList.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  setError: (error: boolean) => void,
+};
+
+export const TodoErrorList: React.FC<Props> = ({ setError }) => {
+  const [isErrorVisible, setIsErrorVisible] = useState(true);
+
+  const hideError = () => {
+    setIsErrorVisible(false);
+
+    setError(false);
+  };
+
+  useEffect(() => {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
```

Comment:
You should clear all timeouts on the component unmount
https://bobbyhadz.com/blog/react-cleartimeout